src/components/ListResultTable/ListResultTable.jsx

```diff
@@ -18,6 +18,8 @@ function ListResultTable(props) {
     resource,
     sort,
     variables,
+    newQuery,
+    submitted,
```

Comment:
Could you refactor "newQuery" to "changeVariables" (makes it more clear...)

Reply:
Done!